app/src/layouts/common/appSidebar/helpAndService/FAQcontent/FAQContent.jsx

```diff
@@ -0,0 +1,136 @@
+import { LinkItem } from 'layouts/common/appSidebar/helpAndService/linkItem/linkItem';
+import { referenceFAQDictionary } from 'common/utils/referenceDictionary';
+import styles from './FAQContent.scss';
+
+const cx = classNames.bind(styles);
+
+export const FAQContent = ({ onFAQOpen, closeNavbar, closePopover }) => {
+  const { formatMessage } = useIntl();
+  onFAQOpen(true);
+  const userId = useSelector(userIdSelector);
+  setFAQOpenStatusTrue(userId);
+  const dispatch = useDispatch();
+
+  const FAQContentItems = [
+    {
+      linkTo: referenceFAQDictionary.configureReportingFAQ,
+      message: messages.configureReporting,
+    },
+    {
+      linkTo: referenceFAQDictionary.improvePerformanceFAQ,
+      message: messages.improvePerformance,
+    },
+    {
+      linkTo: referenceFAQDictionary.configureAnalyzerFAQ,
+      message: messages.configureAnalyzer,
+    },
+    {
+      linkTo: referenceFAQDictionary.integrateJiraFAQ,
+      message: messages.integrateJira,
+    },
+    {
+      linkTo: referenceFAQDictionary.configureCertificateFAQ,
+      message: messages.configureCertificate,
+    },
+    {
+      linkTo: referenceFAQDictionary.fileStorageOptionsFAQ,
+      message: messages.fileStorageOptions,
+    },
+    {
+      linkTo: referenceFAQDictionary.pricingOptionsFAQ,
+      message: messages.pricingOptions,
+    },
+  ];
+  const openModal = () => {
+    closePopover();
+    closeNavbar();
+    dispatch(
+      showModalAction({
+        id: 'requestSupportModal',
+      }),
+    );
+  };
+
+  const furtherAssistanceValues = {
+    support: (
+      <LinkItem
+        link={referenceDictionary.rpEmail}
+        content={formatMessage(messages.supportTeam)}
+        icon={OpenIcon}
+        className={cx('inline-ref')}
+      />
+    ),
+    channel: (
+      <LinkItem
+        link={referenceDictionary.rpSlack}
+        content={formatMessage(messages.slackChannel)}
+        className={cx('inline-ref')}
+        icon={OpenIcon}
+      />
+    ),
+  };
+  return (
+    <>
+      {FAQContentItems.map((contentItem) => {
+        return (
+          <LinkItem
+            link={contentItem.linkTo}
+            content={formatMessage(contentItem.message)}
+            className={cx('menu-item')}
+            onClick={() => {
+              closePopover();
+              closeNavbar();
+            }}
+            key={`faq-${uniqueId()}`}
+          />
+        );
+      })}
+      <p className={cx('assistance')}>
+        {formatMessage(messages.furtherAssistance, {
+          support: () => furtherAssistanceValues.support,
+          channel: () => furtherAssistanceValues.channel,
+        })}
+      </p>
+      <div className={cx('divider')} />
+      <button className={cx('menu-item')} onClick={openModal}>
+        <FormattedMessage
+          id={'Sidebar.services.requestService'}
+          defaultMessage={'Request professional service'}
+        />
```

Comment:
Let's not mix the `FormattedMessage` and `formatMessage` from `useIntl` within single component.
Use `formatMessage` where possible as it is easier to reuse messages defined via `defineMessages`.